gen_wg_config.bash

```diff
@@ -17,12 +17,10 @@ read_config() {
     sswg_log=${config_folder}/sswg.log
 
     baseurl_1="https://api.surfshark.com"
```

Comment:
Can we move them to a json config file so every time a url changes we just change that file?

Reply:
Eid mubarrak.

The urls ? Probably I can. e=Explain how you want it to work and ill see what I can do :;

Reply:
> Can we move them to a json config file so every time a url changes we just change that file?

Thinking about your comment, the way I have it setup now is during the login (which is done during a full run no prarameters) it goes through the urls and will save the succesfull url with the token.json file. So when u run script again with the renew parameter (-n) it will use the same url.